universe/Conditions.cpp

```diff
@@ -26,6 +26,7 @@
 #include "../Empire/EmpireManager.h"
 #include "../Empire/Supply.h"
 
+#include <boost/algorithm/cxx11/all_of.hpp>
```

Comment:
why boost not std?

Reply:
It uses the range style interface instead of a begin, end pair.